### ISSUE TITLE
question about training my own data

### ISSUE THREAD

**Issue by Adarine99** [scaelles/OSVOS-TensorFlow#23]
Hi，
     Appreciate your remarkable work and I want to train my own video data,but how can I get the first Labeled image,Is there any software I can use?Thanks a lot.

**Comment by Adarine99** [comment]
got it ,I'm using the GIMP.

**Comment by scaelles** [comment]
Good to hear that you found a fix yourself! Otherwise, you could also use the [LabelMe](http://labelme.csail.mit.edu/Release3.0/) project.

**Comment by Minotaur-CN** [comment]
1. The label is 3-channel png? or has a palette as VOC?
2. Every sampling video has to train a model?
Thanks!

**Comment by kmaninis** [comment]
1. It's a 1-channel binary image.
2. That's correct. Every video needs online training.

**Comment by Minotaur-CN** [comment]
Thanks for your reply！
I tested some video not from DAVIS, The evaluation results not well, even use on-line training for a long training steps.
How to improve the results? Re-train the parent work?

Thanks!
